indexer-app/test/cmr/indexer/test/data/concepts/keyword_util.clj

```diff
@@ -213,7 +213,27 @@
                       :DetailedVariable "RADAR"}
                      {:Category "SCIENCE CAT 3"
                       :Topic "SCIENCE TOPIC 3"
-                      :Term "SCIENCE TERM 3"}]})
+                      :Term "SCIENCE TERM 3"}]
+   :ArchiveAndDistributionInformation {:FileDistributionInformation [{:FormatType "Native",
+                                                                      :AverageFileSize nil,
+                                                                      :Fees nil,
+                                                                      :Format "netCDF4",
+                                                                      :TotalCollectionFileSize nil,
+                                                                      :TotalCollectionFileSizeBeginDate nil,
+                                                                      :TotalCollectionFileSizeUnit nil,
+                                                                      :Description nil,
+                                                                      :AverageFileSizeUnit nil,
+                                                                      :Media nil}
+                                                                     {:FormatType "Native",
+                                                                      :AverageFileSize nil,
+                                                                      :Fees nil,
+                                                                      :Format "PDF",
+                                                                      :TotalCollectionFileSize nil,
+                                                                      :TotalCollectionFileSizeBeginDate nil,
+                                                                      :TotalCollectionFileSizeUnit nil,
+                                                                      :Description nil,
+                                                                      :AverageFileSizeUnit nil,
+                                                                      :Media nil}]}})
```

Comment:
Nitpick: I think this is easier to read, but feel free to ignore if you feel strongly
```suggestion
   :ArchiveAndDistributionInformation 
     {:FileDistributionInformation 
       [{:FormatType "Native",
         :AverageFileSize nil,
         :Fees nil,
         :Format "netCDF4",
         :TotalCollectionFileSize nil,
         :TotalCollectionFileSizeBeginDate nil,
         :TotalCollectionFileSizeUnit nil,
         :Description nil,
         :AverageFileSizeUnit nil,
         :Media nil}
        {:FormatType "Native",
         :AverageFileSize nil,
         :Fees nil,
         :Format "PDF",
         :TotalCollectionFileSize nil,
         :TotalCollectionFileSizeBeginDate nil,
         :TotalCollectionFileSizeUnit nil,
         :Description nil,
         :AverageFileSizeUnit nil,
         :Media nil}]}})
```